### PR TITLE
Updated cluster limits for 4.2

### PR DESCRIPTION
**Pull request by ahardin-rh**
Preview Build: http://file.rdu.redhat.com/~ahardin/10102019/4-2-limits/scalability_and_performance/planning-your-environment-according-to-object-limits.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@chaitanyaenr PTAL

**Comment by ahardin-rh** [comment]
FYI @ashishkamra Not sure if we still have a JIRA card tracking this

**Comment by ahardin-rh** [comment]
Found the epic: https://jira.coreos.com/browse/PROD-1159

**Comment by ahardin-rh** [comment]
@mffiedler PTAL as well. Thanks!

File: modules/openshift-cluster-limits.adoc [hunk 1]
```diff
@@ -82,8 +82,8 @@ the system has enough CPU, memory, and disk to satisfy the application requireme
 
 |===
 
-In {product-title} {product-version}, half of a CPU core (500 millicore) is now reserved by
-the system compared to {product-title} 3.11 and previous versions.
+In {product-title} {product-version}, half of a CPU core (500 millicore) is
+reserved by the system compared to {product-title} 3.11 and previous versions.
 
-In {product-title} 4.1, the tested node limit has been lowered until scale tests can be run at
+In {product-title} {product-version}, the tested node limit has been lowered until scale tests can be run at
```
Comment (mffiedler) [inline modules/openshift-cluster-limits.adoc]: This may go away based on tests happening right now.  Stay tuned...

Comment (chaitanyaenr) [reply modules/openshift-cluster-limits.adoc]: We can remove the caveat added around the number of nodes per cluster limit.

**Review by chaitanyaenr — APPROVED** [review]
/lgtm

**Pull request merged**